### DISCUSSION TITLE
How to change file name .zip Output Backup Spatie Laravel.

### DISCUSSION THREAD

**Discussion by ibnudirsan** [spatie/laravel-backup#1385] [Q&A]
How to change file name zip
exp: 2021-08-19-02-00-12.zip change 2021-08-19.zip

Thank You...

**Comment by erikn69** [comment]
Try this
```batch
backup:run --filename=2021-08-19
```
Or in kernel
```php
$schedule->command('backup:run --filename='.date('Y-m-d'))->dailyAt('01:00');
```

**Reply by ibnudirsan** [reply]
> Try this
> 
> ```batchfile
> backup:run --filename=2021-08-19
> ```
> 
> Or in kernel
> 
> ```
> $schedule->command('backup:run --filename='.date('Y-m-d'))->dailyAt('01:00');
> ```

Doesn't work, schedule runs but the zip file doesn't exist.

**Reply by ibnudirsan** [reply]
php artisan backup:run --filename=2021-08-19
Starting backup...
Dumping database devatt...
Determining files to backup...
Zipping 965 files and directories...
Backup failed because ZipArchive::addFile(): Invalid or uninitialized Zip object.

**Reply by ibnudirsan** [reply]
$date = date('Y-m-d');
        $schedule->command("backup:run --filename=$date.zip")->dailyAt('01:00');
        

>         SOLVED
>         WORK

        

>  $ php artisan schedule:run
> Running scheduled command: "C:\wamp64\bin\php\php7.2.33\php.exe" "artisan" backup:run --filename=2021-08-19.zip > "NUL" 2>&

**Reply by erikn69** [reply]
>SOLVED
>WORK

Mark this as answered
